chapter_preliminaries/ndarray.md

```diff
@@ -698,6 +699,7 @@ size = lambda a: a.numel()
 reshape = lambda a, *args: a.reshape(*args)
 ones = torch.ones
 zeros = torch.zeros
+tensor = torch.Tensor
```

Comment:
@mli Not sure if this is a very good idea. Defining a tensor is always a `FloatTensor`. We won't be able to use `dtype` for defining the type explicitly with such implementation. I suggest keeping the tensors different for all the 3 frameworks. Besides this may pose problems for readers to understand.